operations/availabilityblocks.md

```diff
@@ -66,220 +68,188 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
+      "AvailabilityBlockId": "aaaa654a4a94-4f96-9efc-86da-bd26d8db",
+      "ResourceCategoryId": "1268c440-21c5-415d-bf58-ac87008b2bda",
+      "FirstTimeUnitStartUtc": "2021-10-14T00:00:00Z",
+      "LastTimeUnitStartUtc": "2021-10-17T00:00:00Z",
+      "UnitCount": 6,
+      "ActivityState": "Active",
+      "UpdatedUtc": "2021-10-21T13:32:32Z"
+    }
+  ],
+  "Cursor": "aaaa654a4a94-4f96-9efc-86da-bd26d8db"
 }
 ```
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `AvailabilityBlocks` | array of [Availability block](#availability-block) | optional | Availability blocks. |
-| `ServiceOrders` | array of [Reservation](reservations.md#reservation-ver-2017-04-12) | optional | Service orders (for example reservations) linked to availability blocks. |
+| `AvailabilityBlocks` | array of [Availability block](availabilityblocks.md#availability-block) | optional | Availability blocks. |
+| `ServiceOrders` | array of [Reservation (ver 2017-04-12)](reservations.md#reservation-ver-2017-04-12) | optional | Service orders (for example reservations) linked to availability blocks. |
 | `Adjustments` | array of [Availability adjustment](availabilityadjustments.md#availability-adjustment) | optional | Availability adjustments of availability blocks. |
+| `Rates` | array of [Rate](rates.md#rate) | optional | `Rates` assigned to the block. |
 | `Cursor` | string | optional | Unique identifier of the item one newer in time order than the items to be returned. If Cursor is not specified, i.e. null, then the latest or most recent items will be returned. |
 
 #### Availability block
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Id` | string | required | Unique identifier of the availability block. |
+| `EnterpriseId` | string | required | Unique identifier of the [enterprise](enterprises.md#enterprise). |
 | `ServiceId` | string | required | Unique identifier of the [Service](services.md#service) the block is assigned to. |
 | `RateId` | string | required | Unique identifier of the [Rate](rates.md#rate) the block is assigned to. |
 | `VoucherId` | string | optional | Unique identifier of the [Voucher](vouchers.md#voucher) used to access specified private [Rate](rates.md#rate). |
 | `BookerId` | string | optional | Unique identifier of the [Customer](customers.md#customer) on whose behalf the block was made. |
 | `CompanyId` | string | optional | Unique identifier of the [Company](companies.md#company) linked to the block. |
-| `Budget` | [Currency value](accountingitems.md#currency-value) | optional | The tentative budget for the total price of reservations in the block. |
-| `State` | string [Availability block state](#availability-block-state) | required | State of the availability block. |
-| `ReservationPurpose` | string [Reservation purpose](reservations.md#reservation-purpose) | optional | The purpose of the block. |
+| `TravelAgencyId` | string | optional | Unique identifier of the `Travel agency`. |
```

Comment:
Technically there is no entity `Travel agency`. `Travel Agency` is a company that has assigned travel agency contracts 🤔
I would go with same description as here https://mews-systems.gitbook.io/connector-api/operations/vouchers#voucher for `TravelAgencyId` 🚀